lib/rsolr/document.rb

```diff
@@ -38,7 +38,11 @@ def field_by_name(name)
     def add_field(name, values, options = {})
       wrap(values).each do |v|
         next if v.nil?
-        @fields << RSolr::Field.instance(options.merge({:name=>name}), v)
+        if name.to_s == '_childDocuments_'
```

Comment:
Should this stiring be frozen or should we add the frozen string pragma to the file?